articles/openshift/howto-deploy-java-openliberty-app.md

```diff
@@ -92,12 +95,14 @@ To access the built-in container image registry provided by the ARO 4 cluster, a
 
 OpenShift Container Platform provides a built-in container image registry which runs as a standard workload on the ARO 4 cluster. The registry provides an out of the box solution for users to manage the images that run their workloads, and runs on top of the existing cluster infrastructure. Follow the instructions below to enable the access of the built-in registry.
 
-1. Log in to the OpenShift CLI using the token retrieved from the web console with a user from Azure AD.
+1. Log in to the OpenShift web console from your browser using the credentials of an Azure AD user.
    > [!NOTE]
    > Write down the user name and password for this Azure AD user, as it will be used as **the administrator** for the demo project throughout this guide and subsequent ones.
-2. Run `oc whoami` in the console and write down its output as **\<aad-user>**.
-3. Log in to the OpenShift CLI using the token retrieved from the web console with the `kubeadmin` credentials.
-4. Execute the following commands to enable the access to the built-in registry for the **aad-user**.
+2. [Log in to the OpenShift CLI with the token](#log-in-to-the-openshift-cli-with-the-token).
```

Comment:
Because item 2 in the list is a link to the instructions for item 1 it creates the impression there are two steps, when actually there's just one.  I would combine these lines to something like:
```
1. Log in to the OpenShift web console from your browser using the credentials of an Azure AD user (for help with how to log in, see [Log in to the OpenShift CLI with the token](#log-in-to-the-openshift-cli-with-the-token) ).
```

I would also do the same for steps 4 and 5.

Reply:
Thanks @gcharters for your quick response. Item 1 is to "Log in to the OpenShift **web console**", but item 2 is to Log in to the OpenShift **CLI** with the token, there're indeed two steps. :)